src/cloud/nabu-casa-account/mfa-disable.md

```diff
@@ -8,6 +8,8 @@ zendesk:
 
 Follow these steps to disable multi-factor authentication (MFA) for your Home Assistant Cloud account.
 
+- Note that once you have disabled multi-factor authentication (MFA), you can not enable it again. The function to enable it is not currently available.
```

Comment:
As the other page has more details, it would be good to link to that here, as this offers no explanation.